gcloud-java-storage/src/main/java/com/google/gcloud/storage/testing/FakeStorageRpc.java

```diff
@@ -57,12 +56,6 @@ public FakeStorageRpc(boolean throwIfOption) {
     this.throwIfOption = throwIfOption;
   }
 
-  // remove all files
-  void reset() {
-    stuff = new HashMap<>();
-    contents = new HashMap<>();
-  }
-
   @Override
   public Bucket create(Bucket bucket, Map<Option, ?> options) throws StorageException {
     throw new UnsupportedOperationException();
```

Comment:
Any reason for the fully qualified `com.google.common.io.ByteStreams` instead of adding an import?